src/app.rs

```diff
@@ -583,6 +584,11 @@ lazy_static! {
               only clears the default type definitions that are found inside \
               of ripgrep.\n\nNote that this MUST be passed to every \
               invocation of ripgrep. Type settings are NOT persisted.");
+        doc!(h, "search-zip",
+             "Search through compressed files.",
+             "Search through compressed files. Currently gz, bz2, xz, and \
+              lzma files are supported. This options expects the uncompression \
```

Comment:
Same comment about 'on your `PATH`', but also i think 'uncompression' should be 'decompression' here